src/ui/components/Library/Team/View/TimeFilterContextRoot.tsx

```diff
@@ -55,3 +59,14 @@ export function TimeFilterContextRoot({ children }: { children: ReactNode }) {
     </TimeFilterContext.Provider>
   );
 }
+
+export function withinTeamRetentionLimit(
+  team: Workspace | typeof MY_LIBRARY_TEAM | null | undefined,
+  days: number
+) {
+  if (!team || !("retentionLimit" in team)) {
```

Comment:
I don't think the `"retentionLimit" in team` is required since you've added that key to `MY_LIBRARY_TEAM` too